cmd/root.go

```diff
@@ -35,9 +36,18 @@ func NewRootCommand(
 		Long:    "LaunchDarkly CLI to control your feature flags",
 		Version: version,
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			// disable required flags when running certain commands, not a flag
-			if cmd.Name() == "help" || cmd.Parent().Name() == "completion" {
-				cmd.DisableFlagParsing = true
+			// disable required flags when running certain commands
+			for _, name := range []string{
```

Comment:
This also fixes a bug where running `ldcli completion bash` would would flags which it shouldn't.